dashboard/app/models/lesson.rb

```diff
@@ -372,11 +372,25 @@ def summarize_for_lesson_show(user)
     }
   end
 
-  def summarize_for_lesson_dropdown
+  def summarize_for_student_lesson_plan
+    all_resources = resources_for_lesson_plan(false)
+    {
+      unit: script.summarize_for_lesson_show(true),
+      position: relative_position,
+      key: key,
+      displayName: localized_name,
+      overview: student_overview || '',
+      announcements: (announcements || []).select {|announcement| announcement['visibility'] != "Teacher-only"},
+      resources: (all_resources['Student'] || []).concat(all_resources['All'] || []),
```

Comment:
I think you could do `all_resources.except('Teacher')` here

Reply:
That gives it to me in object form aka {"Student": [], "All": []}. I thought it would be better to do the concatenation on the ruby side but I can do it on the frontend if you think that's better.

Reply:
Ahh makes sense. I think it's fine this way :)